gadgets/top_file/README.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: top_file
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# top_file
+
+The top_file gadget reports periodically the read/write activity by file.
+
+## Getting started
+
+Running the gadget:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/top_file:latest [flags]
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ ig run ghcr.io/inspektor-gadget/gadget/top_file:latest [flags]
+        ```
+    </TabItem>
+</Tabs>
+
+## Flags
+
+TODO: Add flags added by operators.
+
+### `--all-files`
+Trace all files types. By default, only regular files are traced
+
+Default value: "false"
+
+### `--pid`
+Show only events generated by process with this PID
+
+Default value: "0"
+
+## Guide
+
+This example shows how to use this gadget. To do so, we will run an application
+that performs some disk I/O activity.
+
+Before starting our application, let's start our `top_file` gadget to be sure it
+captures all the events from the beginning:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl gadget run top_file --pod mypod
```

Comment:
any reason not to use the :latest suffix and whole name as done in other gadgets?

Reply:
I added it. I haven't noticed other gadgets were using it. However, it's unnecessary, `latest` will be added automatically by IG, like docker does and people are used to do, don't you think?

Reply:
I'd prefer to keep the :latest to make it obvious it's an OCI image. We can discuss it more and uniform all guides to follow the same pattern.